Modules/SystemFolder/classes/Setup/class.ilInstallationInformationStoredObjective.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/* Copyright (c) 2019 Richard Klees <richard.klees@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+use ILIAS\Setup;
+
+class ilInstallationInformationStoredObjective implements Setup\Objective {
```

Comment:
Please add a comment here and in similar places that explain the purpose of the objective.